source/index.ts

```diff
@@ -247,3 +247,7 @@ Object.defineProperties(is, {
 });
 
 export default is; // tslint:disable-line:no-default-export
+
+// For CommonJS default export support
+module.exports = is;
+module.exports.default = is;
```

Comment:
Doesn't `export default is` get compiled to this?

Reply:
No, it get's compiled to `exports.default = is`...

Reply:
Ah, too bad.